python/tk_multi_publish2/base_hooks/publish_plugin.py

```diff
@@ -415,12 +415,13 @@ def finalize(self, settings, item):
     # allows clients to write their own publish plugins while deferring custom
     # UI settings implementations until needed.
 
-    def create_settings_widget(self, parent):
+    def create_settings_widget(self, parent, items):
```

Comment:
Perhaps this should be a kwarg, in case a deriving class  that has not been updated calls the base without passing `items`?
Same goes for `set_ui_settings`

Reply:
Could it be `None` instead? I dislike backward comp via `kwargs`.

Reply:
When you say `None`, do you mean use a kwarg with a default of `None`, `items=None`?

Reply:
I meant `items=None`. Do you think that could work?